content/designers/theme-objects/ddrmenu/index.md

```diff
@@ -11,7 +11,8 @@ links: ["[DNN Wiki: DotNetNuke Skins](https://www.dnnsoftware.com/wiki/dotnetnuk
 
 # DDRMenu Overview  
 
-Originally built and maintained by [Mark Allan @MarkXA](https://twitter.com/markxa), the DDRMenu is a robut menu system that is installed and used by default in DNN since version 6.0.  
+Originally built and maintained by [Mark Allan @MarkXA](https://twitter.com/markxa), the DDRMenu is a robust menu system that is installed and used by default in DNN since version 6.0.
+With the right Template and Styling any menu style can be created, with DDR menu you can use [TOKEN], XSLT or Razor templates.
```

Comment:
`style can be created, with DDR menu` should be `style can be created. with DDR menu`